learn/advanced/updating.md

```diff
@@ -8,6 +8,8 @@ Currently, Meilisearch databases are only compatible with the version of Meilise
 
 If you're updating your Meilisearch instance on cloud platforms like DigitalOcean, AWS, or GCP, ensure that you can connect to your cloud instance via SSH. Depending on the user you are connecting with (root, admin, etc.), you may need to prefix some commands with `sudo`.
 
+[Refer to the Version-specific update warnings sections for more details on upgrading from specific versions.](#version-specific-update-warnings)
```

Comment:
```suggestion
If migrating to the latest version of Meilisearch will cause you to skip multiple versions, this may require changes to your codebase. [Refer to our version-specific update warnings for more details](#version-specific-warnings).
```